libs/single-spa-angular/webpack/index.ts

```diff
@@ -43,9 +43,9 @@ export default (config: any, options?: Options, extraOptions?: DefaultExtraOptio
     module: {
       rules: [
         {
-          parser: {
+          /*parser: {
```

Comment:
Does angular 12 use webpack 5? The parser system false is required in webpack 4, but not 5.

Also, can you delete this code rather than comment it out?

Reply:
At least when I tried to get it working in fresh demo projects or our MFEs I always got that error, even when explicitly declaring Webpack 4 in the dependencies. I looked at yarn.lock and saw that @angular-builders/custom-webpack v12 forced Webpack 5 on its dependencies. So I assumed this was the error, came here looking for issues and found #379.

I'm not at work today so I can't show you the yarn.lock right now.

Reply:
yarn.lock:
https://github.com/survivejs/webpack-merge/blob/c658e1904a1ebc440fdda6fc88fca1332fb02ea9/package.json#L29
![image](https://user-images.githubusercontent.com/15238295/126942394-642be216-7e53-464d-a96a-ce6d5319356c.png)
@angular-devkit/build-angular also has v5 as dependency.

Error:
![image](https://user-images.githubusercontent.com/15238295/126943074-e5f2248a-7bc5-44db-a517-2f95a3812f75.png)

package.json:
![image](https://user-images.githubusercontent.com/15238295/126949612-3a2253e7-a9d3-4c11-abfd-ffe247886c70.png)

Reply:
The first screenshot shows the webpack-merge version for `@angular/custom-webpack`, not the webpack version. Angular ignores any webpack versions in the package.json. Instead, it uses the version specified by `@angular-devkit/build-angular`.

I tested it out and confirmed that it does use webpack 5.

Reply:
I didn't knew that. Glad it could be merged. Thank you, Joel!